### DISCUSSION TITLE
Strange download starts causing problems

### DISCUSSION THREAD

**Discussion by federicomameli** [cefsharp/CefSharp#4180] [Q&A]
Hi,
after upgrading to version 102, we found in the Cef log the following line:
VERBOSE1:background_downloader_win.cc(445)] Starting BITS download for: http://edgedl.me.gvt1.com/edgedl/chromewebstore/L2Nocm9tZV9leHRlbnNpb24vYmxvYnMvZjE0QUFYTUR2NXNIakJsbE5jbXNrUkdfQQ/4.10.2391.0_oimompecagnajdejgnnjijobebaeigek.crx
We suspect during this download the component is in someway less responsive or at least handler call might be delayed causing problems to our application.
What is this download for? is there any way to avoid it and make sure this is causing the problems we are experiencing right now?
Thank you

**Comment by amaitland** [comment]
I'd suggest moving the conversation over to the CEF support forum at https://magpcss.org/ceforum/viewforum.php?f=6 

Please post a link back here for reference. 

If you have problems signing up follow the instructions at https://bitbucket.org/chromiumembedded/cef/issues/2962/unable-to-register-in-your-forums

**Reply by amaitland** [reply]
For reference I believe it's the Widevine CDM component which is an expected download. I've never seen it cause a problem before.

https://chromium.googlesource.com/chromium/chromium/+/trunk/chrome/browser/component_updater/widevine_cdm_component_installer.cc#47

For background https://www.bleepingcomputer.com/news/security/what-are-these-suspicious-google-gvt1com-urls/

**Reply by amaitland** [reply]
Looks like you posted https://magpcss.org/ceforum/viewtopic.php?f=6&t=19129

In future please post a link to any follow up discussion on ceforum so others can benefit.
